### DISCUSSION TITLE
Build warning on arm64

### DISCUSSION THREAD

**Discussion by gatortux** [docker-mailserver/docker-mailserver#4116] [Q&A] [answered]
I followed the build docker image instructions on the DMS website to keep a few packages up to date. One of my servers I run on is an arm64 server and that particular build ends with this warning:

 1 warning found (use --debug to expand):
 - InvalidBaseImagePlatform: Base image docker.io/clamav/clamav:latest was pulled with platform "linux/amd64", expected "linux/arm64" for current build (line 65)

Is this something I should worry about? Here is my build command:

docker build --tag mailserver/custom-mailserver:latest --build-arg DMS_RELEASE=latest .

I tried adding the platform parameter to the build and got the same result. 

docker build --platform linux/arm64 --tag mailserver/custom-mailserver:latest --build-arg DMS_RELEASE=latest .

Adding --debug as the message suggests results in an invalid parameter message.

**Comment by casperklein** [comment, marked as answer]
> Is this something I should worry about? Here is my build command:

No. It's just a warning that you can ignore. [There is no official ARM image available for ClamAV](https://hub.docker.com/r/clamav/clamav/tags). While building, we copy the latest ClamAV DB from the (AMD64) image. But that's not a problem. We don't copy any binarys, so the platform doesn't matter.
